src/UserFrontend.php

```diff
@@ -61,13 +61,22 @@ public static function login_head() {
    */
   public static function login_form_defaults(array $args) {
     $args['value_remember'] = TRUE;
+    $args['remember'] = FALSE;
     return $args;
   }
 
   /**
    * @implements login_enqueue_scripts
    */
   public static function login_enqueue_scripts() {
+    echo <<<EOD
+<style>
+#loginform .forgetmenot {
+  display: none;
+}
+</style>
```

Comment:
Agree. I did it because I was not sure about the correct beahviour. 
Is not the existing CSS below printed only if a `login.css` file not exists?

```
if (file_exists(get_stylesheet_directory() . '/dist/styles/login.css')) {
…
} 
else {
  echo <<<EOD
<style>
…
}
```

So if I put the code inside that style we should, in SVC case, add same CSS rule also to `login.css`. Is it correct?